modules/light-clients/06-solomachine/light_client_module_test.go

```diff
@@ -1,18 +1,797 @@
 package solomachine_test
 
 import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
 	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
+	commitmenttypes "github.com/cosmos/ibc-go/v8/modules/core/23-commitment/types"
 	host "github.com/cosmos/ibc-go/v8/modules/core/24-host"
+	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
 	"github.com/cosmos/ibc-go/v8/modules/core/exported"
 	solomachine "github.com/cosmos/ibc-go/v8/modules/light-clients/06-solomachine"
 	ibctesting "github.com/cosmos/ibc-go/v8/testing"
+	ibcmock "github.com/cosmos/ibc-go/v8/testing/mock"
 )
 
 const (
 	smClientID   = "06-solomachine-100"
 	wasmClientID = "08-wasm-0"
 )
 
+func (suite *SoloMachineTestSuite) TestStatus() {
+	clientID := suite.chainA.App.GetIBCKeeper().ClientKeeper.GenerateClientIdentifier(suite.chainA.GetContext(), exported.Solomachine)
+	clientState := suite.solomachine.ClientState()
+
+	// Set a client state in store.
+	suite.chainA.App.GetIBCKeeper().ClientKeeper.SetClientState(suite.chainA.GetContext(), clientID, clientState)
```

Comment:
Is it necessary to set a new client state? Cannot you use the client state that you retrieve with `suite.solomachine.ClientState()` and its associated client ID with `suite.solomachine.ClientID`?